nexus-sdk/src/main/java/io/nexusrpc/handler/OperationStartDetails.java

```diff
@@ -87,11 +101,20 @@ public Builder setRequestId(String requestId) {
       return this;
     }
 
+    /** Set links. */
+    public Builder setLinks(List<Link> links) {
```

Comment:
The way we have done collections in builders in the Nexus SDK thus far is to have a list they can add to instead of accepting a full list. For example, see `ServiceDefinition.Builder` and `OperationContext.Builder` and `ServiceImplInstance.Builder` and even here for callback headers. I think it's a cleaner model than accepting a collection (but we can discuss changing in a consistent way).

Reply:
Sure we should be consistent here

Reply:
👍, I'm ready to mark approved once this part is changed

Reply:
Done here https://github.com/nexus-rpc/sdk-java/pull/12/commits/a49c3429295d197de05d673787897f782be4ef1d

Reply:
I think that commit may be an orphan somehow? For some reason the latest for this PR is still showing `setLinks` as is the latest view of this file on the branch associated w/ the PR: https://github.com/Quinn-With-Two-Ns/nexus-sdk-java/blob/issue-11/nexus-sdk/src/main/java/io/nexusrpc/handler/OperationStartDetails.java

Reply:
Not sure, if you look at my branch the commit is there https://github.com/Quinn-With-Two-Ns/nexus-sdk-java/tree/issue-11

Reply:
oh I see you commented on `OperationStartDetails`, but I only changed `OperationStartResult` my bad